.github/workflows/builds.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       - name: Install package
         run: |
           python -m pip install --upgrade pip
+          pip install -r requirements.txt
```

Comment:
That line isn't needed: the next one will install all deps listed in `setup.py` along with the lib. And the latest pypi release of pylocron is a valid one, so I'd suggest to remove this